### PR TITLE
Fix/update required field docstring

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, I updated role_id to required_field and write docstring in random_id function, added comments in update_account function
#42

### CODE REVIEW

File: myproject/supermarket_manager/views.py [hunk 1]
```diff
@@ -485,6 +485,8 @@ def update_account(request, account_id):
         data = json.loads(request.body)
 
         if request.method == "PUT":
+            # Update account information but do not change password
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/views.py]: Should comment above the `if` statement.

File: myproject/supermarket_manager/views.py [hunk 2]
```diff
@@ -494,6 +496,8 @@ def update_account(request, account_id):
             account.status = data.get("status", account.status)
 
         elif request.method == "PATCH":
+            # Change password
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/views.py]: Should comment above the `elif` statement.

File: myproject/supermarket_manager/models.py [hunk 3]
```diff
@@ -49,13 +49,9 @@ class Role(models.Model):
...
         """
-        Return a string representation of the Role object.
+        Return a string representation of the Permission object.
         """
-
-        if isinstance(self.role_name, str):
-            return self.role_name
-        else:
-            return str(self.role_name)
+        return str(self.permission_name)
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/models.py]: Please check again.

File: myproject/supermarket_manager/views.py (hunk 2 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

File: myproject/supermarket_manager/models.py (hunk 3 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/models.py]: Done

File: myproject/supermarket_manager/views.py (hunk 1 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

File: myproject/supermarket_manager/models.py [hunk 4]
```diff
@@ -49,13 +49,9 @@ class Role(models.Model):
 
     def __str__(self):
         """
-        Return a string representation of the Role object.
+        Return a string representation of the Permission object.
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/models.py]: Please check again.

**Pull request merged**